src/helpers/runNotificationListener.ts

```diff
@@ -4,28 +4,44 @@ import sendAppleNotification from '@/helpers/sendAppleNotification'
 import sendGoogleNotification from '@/helpers/sendGoogleNotification'
 
 const apnRegex = /^[a-f0-9]{64}$/
+const notificationInterval = 30 * 60 * 1000 // 30 minutes in milliseconds
 
 const rootFeeds = {
   1: 't/startups',
   2: 't/ketlTeam',
 } as { [key: number]: string }
```

Comment:
Won't it better to fetch feeds from the chain, because the order can changed or smt else

Reply:
maybe later